src/commands/nfts/getJWT.ts

```diff
@@ -0,0 +1,50 @@
+import { Account, Nevermined } from '@nevermined-io/sdk'
+import { StatusCodes } from '../../utils'
+import { ExecutionOutput } from '../../models/ExecutionOutput'
+import chalk from 'chalk'
+import { Logger } from 'log4js'
+import { ConfigEntry } from '../../models/ConfigDefinition'
+
+export const getJWT = async (
+  nvm: Nevermined,
+  userAccount: Account,
+  argv: any,
+  config: ConfigEntry,
+  logger: Logger
+): Promise<ExecutionOutput> => {
+  const { did } = argv
+
+  logger.info(
+    chalk.dim(`Gets JWT Access Token ${chalk.whiteBright(did)}`)
+  )
+
+  let ddo
+  
+  try {
+    ddo = await nvm.assets.resolve(did)
```

Comment:
is there a need to resolve the did?

Reply:
it's just for failing if the DID doesn't exist